Tests/SwiftParserTest/RegexLiteralTests.swift

```diff
@@ -76,7 +79,10 @@ final class RegexLiteralTests: XCTestCase {
       """#,
       diagnostics: [
         DiagnosticSpec(message: "expected '/' to end regex literal", fixIts: ["insert '/'"])
-      ]
+      ],
+      fixedSource: #"""
+        //
```

Comment:
We shouldn’t be offering this fix since this results in a line comment instead of a regex literal.